.travis.yml

```diff
@@ -88,7 +89,20 @@ script:
       ./build/sbt -Dspark.version=$SPARK_VERSION
```

Comment:
How do you feel about using the travis matrix to run the style tests as separate tasks. For example we could have a matrix that looks something like:

```
env:
  global:
      ...
  matrix:
    - SUITE=python, PYTHON_VERSION=2.7.13
    - SUITE=python, PYTHON_VERSION=3.5.1
    - SUITE=python, PYTHON_VERSION=3.6.2
    - SUITE=style, PYTHON_VERSION=2.7.13
    - SUITE=style, PYTHON_VERSION=3.6.2
    - SUITE=scala, PYTHON_VERSION=3.6.2
```

`run.py` can then choose the right tests to run.